sources/osg/KdTreeSphereIntersector.js

```diff
@@ -1,17 +1,15 @@
 'use strict';
 var MACROUTILS = require( 'osg/Utils' );
 var vec3 = require( 'osg/glMatrix' ).vec3;
-var KdTreeRayIntersector = require( 'osg/KdTreeRayIntersector' );
+var KdTreeLineSegmentIntersector = require( 'osg/KdTreeLineSegmentIntersector' );
```

Comment:
For this I agree, because users could be using directly this intersector, and it is not a ray is a linesegment. So OK.